spring-integration-ip/src/main/java/org/springframework/integration/ip/dsl/TcpOutboundGatewaySpec.java

```diff
@@ -114,6 +114,11 @@ public TcpOutboundGatewaySpec async(boolean async) {
 		return this;
 	}
 
+	public TcpOutboundGatewaySpec unsolictedMessageChannelName(String channelName) {
```

Comment:
Looks like you have there this setter as well `setUnsolicitedMessageChannel(MessageChannel unsolicitedMessageChannel)`. Don't we need to have similar option here on DSL level?
Thanks